score/resource.go

```diff
@@ -0,0 +1,12 @@
+package score
+
+import (
+	"github.com/bitmark-inc/autonomy-api/schema"
+)
+
+func ResourceScore(count int64, sum float64, rating schema.RatingResource) (int64, float64, float64) {
```

Comment:
firstRating = (userA, resourceX, 3)
secondRating = (userA, resourceX, 2)

If a user updates her rating for a resource, then the final rating should be 2 instead of (3+2)/2 = 2.5.

Reply:
wiil do